src/deploy/invokeAutomationTests.js

```diff
@@ -0,0 +1,25 @@
+const fetch = require('node-fetch');
+
+function invokeAutomationTests({ jenkinsUrl, devicePlatforms, whakapapaBranchName, skylineBranchName, skylineS3BranchDirName, buildJobUrl, sourceRepo }) {
+    const url = jenkinsUrl || "https://ctv-jenkins-02.truex.com/generic-webhook-trigger/invoke";
+
+    return fetch(url, {
+        method: "POST",
+        headers: {
+            "Content-Type": "application/json",
+            "token": "TAR-HTML5_UAT",
+        },
+        body: JSON.stringify({
+            "devicePlatforms": devicePlatforms || "Unknown,Tizen,LG,XboxOne,FireTV,AndroidTV,Vizio,Comcast",
+            "whakapapaBranchName": whakapapaBranchName,
```

Comment:
default to develop if missing
is this actually needed for the caller to know?

Reply:
can you file a ticket for wakapapa to support an "All" value so we don't have to maintain the known set here
e.g. what happens when we can start including PS4 and PS5?